doc/Quick-Start-Guide.rst

```diff
@@ -6,85 +6,98 @@ This guide provides information on how to get Motr ready and test it. After foll
 
-2. If using lnet as the transport, check the Lustre network interface configuration::
+   On Ubuntu / Debian OSes::
 
-    sudo vi /etc/modprobe.d/lnet.conf
+    sudo apt install devscripts equivs # enable mk-build-deps
+    cd cortx-motr && sudo mk-build-deps --install debian/control
 
-   Use ``ip a`` command to get a list of network interfaces.
-   Then modify ``lnet.conf`` to use one of the listed network interfaces.
-   After this run::
+2. Choose the networking transport.
 
-    sudo modprobe lnet
-    sudo lctl list_nids
+   Currently there are two options: Lustre LNet and libfabric.
+   LNet is the legacy transport in Motr used for years. libfabric
+   was added recently, but it's the mainline transport now in CORTX.
+   If libfabric is installed in the system and detected by configure,
+   Motr will be built to work with libfabric only (no matter whether
+   LNet is installed or not).
 
-   Make sure that libfabric package is not installed.
-   Please refer the following document for un-installation of libfabric package.
-   https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/711230113/Libfabric+setup+and+using+libfabric+with+motr#Uninstalling-libfabric-package ::
+   In LNet case, check the network interface configuration at
+   ``/etc/modprobe.d/lnet.conf``, make sure the interface name is correct
+   there and matches with the one you have in your system. Here is an
+   example configuration line::
 
-    fi_info --version
-    bash: fi_info: command not found
+    options lnet networks=tcp(eth0) config_on_load=1
 
-3. If using libfabric as the transport, check the Libfabric network interface configuration::
+   Notes:
 
-    sudo vi /etc/libfab.conf
+   - LNet transport option requires support from Motr kernel module and
+     it implies the kernel version dependency. Thus, this option won't fit
+     for contrainer-based deployments. For container-based deployments,
+     consider using libfabric, and use ``--with-user-mode-only`` configure
+     option during the build (see below).
+   - Make sure libfabric package is not installed for LNet-transport builds.
+     If it is installed, uninstall it manually.
 
-   Use ``ip a`` command to get a list of network interfaces.
-   Then modify ``libfab.conf`` to use one of the listed network interfaces.
-   Verify the libfab.conf file contents::
+   In libfabric case, download and install the packages from:
 
-    cat /etc/libfab.conf
-    networks=tcp(eth1)
+   - https://github.com/Seagate/cortx/releases/download/build-dependencies/libfabric-1.11.2-1.el7.x86_64.rpm
+   - https://github.com/Seagate/cortx/releases/download/build-dependencies/libfabric-devel-1.11.2-1.el7.x86_64.rpm
 
-   Please refer the below document for installation.
+   Currently, there is a performance issue with default libfabric versions
+   provided by Linux distributions on tcp. That's why we build customised
+   version of the library. Hopefully, the issue will be resolved soon.
```

Comment:
>Hopefully, the issue will be resolved soon.
 > For more information about our changes to libfabric refer to
 Instead of above statement, it can be,
 Currently only custom libfabric package only is compatible and support for default 
 libfabric will be added soon.

Reply:
@madhavemuri, by "compatible" you probably mean the panic issue? I didn't want to mention it here at all, as it might be fixed very soon.

The issue mentioned here is the performance one.

Reply:
Hopefully, the issue will be resolved soon.
 > I mean there is no clarity on who will resolve this issue.